modules/fluid_properties/include/materials/CO2FluidPropertiesTestMaterial.h

```diff
@@ -0,0 +1,61 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef CO2FLUIDPROPERTIESTESTMATERIAL_H
+#define CO2FLUIDPROPERTIESTESTMATERIAL_H
+
+#include "CO2FluidProperties.h"
+#include "Material.h"
+
+class CO2FluidPropertiesTestMaterial;
+
+template<>
+InputParameters validParams<CO2FluidPropertiesTestMaterial>();
+
+/**
+ * Material for testing the CO2FluidProperties.
+ * Note: This material is for testing purposes only and shouldn't be
+ * used for actual simulations. It provides access to internal functions
+ * as well as the functions provided by the base FluidProperties class
+ */
+class CO2FluidPropertiesTestMaterial : public Material
+{
+public:
+  CO2FluidPropertiesTestMaterial(const InputParameters & parameters);
+  virtual ~CO2FluidPropertiesTestMaterial();
+
+protected:
+  virtual void computeQpProperties();
+
+  /// Pressure (Pa)
+  const VariableValue & _pressure;
+  /// Temperature (K)
+  const VariableValue & _temperature;
+  /// Sublimation pressure (Pa)
+  MaterialProperty<Real> & _psub;
+  /// Melting pressure (Pa)
+  MaterialProperty<Real> & _pmelt;
+  /// Vapour pressure (Pa)
```

Comment:
Let's go with the AE spelling: vapor. I am putting the comment only here, but it is on several other lines below